### ISSUE TITLE
Add/edit validations not longer working

### ISSUE THREAD

**Issue by trob** [joomlahenk/fabrik#67]
I'm pretty confused meanwhile.

Add/edit validations in element settings was working until (including) https://github.com/joomlahenk/fabrik/commit/22fb225c7121ac4eedf96a539ed109b587c30e39

Now if trying to add a new validation  it doesn't populate the validation plugin dropdown and existing validations won't finish loading.

It doesn't matter if administrator\components\com_fabrik\models\element.php will use 
$plugin->getJForm();
or
$plugin->getJform();

components\com_fabrik\models\plugin.php was updated with
function getJform(); and self::getJform();
so I assume element.php should be $plugin->getJform();

but it doesn't seem to be related to the validation population (even if resetting plugin.php and element.php to the old working getForm() doesn't change anything)

**Comment by joomlahenk** [comment]
Validations are not installed as far as I know, at least they should not so the dropdown does not pupulate indeed. Till now I didn't try any validation since I have no validation plugin updated yet. Sure some updates will be needed before validations will work anyway and not just in the plugin itself, but most in the component. It's not on my priority right now, but if you are willing to give it a start?

**Comment by joomlahenk** [comment]
Maybe change the issue name to "Update validations" and we can also add "Update javascript"

**Comment by trob** [comment]
You can easily install all plugins via "Discover" if you have a core Fabrik installed.
That's what I did with additional elements, cron system, cron plugins, validation plugins...

Validations were working, I tested with notempty and php, even two validations in one element.
The validation plugin dropdown was populated (surely only after installing the plugins).
You could add the validations, save the element, got the correct failed messages when trying to safe a record etc.
The existing elements with these validations are still working, but these elements can not longer be edited.

**Comment by trob** [comment]
`The function name in both places should be getJForm.`Yup, but that doesn't help.

What about
JForm::addFormPath(JPATH_SITE..  in administrator\components\com_fabrik\models\element.php (line 659)
and
Form::addFormPath($path); in components\com_fabrik\models\plugin.php (line 282)

`If you run the build process` Silly question: how?

**Comment by joomlahenk** [comment]
run the build process: just do a download.zip fron GH -> unzip and goto to the plugin you want to add -> zip that one -> install in J!4 -> and voila

**Comment by skurvish** [comment]
On the whatabout, in both of those cases it should be Form as they are calling a function related to the Form class. the getJForm function is a special case to get the form associated with a particular plugin.

I just now published a document on the Wiki on how to do a build.

**Comment by trob** [comment]
` just do a download.zip fron GH` :) thanks (big enough so). Although it should be the same to install via discover or via single zips

**Comment by trob** [comment]
It's not only validation plugins, I opened a new issue https://github.com/joomlahenk/fabrik/issues/78
